### PR TITLE
Jetpack Compose 1.7.0-beta06

### PR DESCRIPTION
**Pull request by igordmn**

### CODE REVIEW

**Review by mazunin-v-jb — APPROVED** [review]

File: compose/ui/ui-graphics/src/androidMain/kotlin/androidx/compose/ui/graphics/layer/AndroidGraphicsLayer.android.kt [hunk 1]
```diff
@@ -760,9 +761,11 @@ internal constructor(
      * @sample androidx.compose.ui.graphics.samples.GraphicsLayerRoundRectOutline
      */
     actual fun setRoundRectOutline(topLeft: Offset, size: Size, cornerRadius: Float) {
-        if (this.roundRectOutlineTopLeft != topLeft ||
-            this.roundRectOutlineSize != size ||
-            this.roundRectCornerRadius != cornerRadius
+        if (
+            this.roundRectOutlineTopLeft != topLeft ||
```
Comment (MatkovIvan) [inline compose/ui/ui-graphics/src/androidMain/kotlin/androidx/compose/ui/graphics/layer/AndroidGraphicsLayer.android.kt:765]: Shouldn't we copy this to `SkiaGraphicsLayer.skiko.kt`? If you're not sure, please add TODO for me

File: compose/ui/ui/src/commonMain/kotlin/androidx/compose/ui/focus/FocusOwnerImpl.kt [hunk 2]
```diff
@@ -264,7 +265,8 @@ internal class FocusOwnerImpl(
 
         val activeFocusTarget = rootFocusNode.findActiveFocusNode()
         val focusedKeyInputNode = activeFocusTarget?.lastLocalKeyInputNode()
-            ?: activeFocusTarget?.nearestAncestorIncludingSelf(Nodes.KeyInput)?.node
+                ?: activeFocusTarget?.nearestAncestorIncludingSelf(Nodes.KeyInput)?.node
+                ?: rootFocusNode.nearestAncestor(Nodes.KeyInput)?.node
```
Comment (MatkovIvan) [inline compose/ui/ui/src/commonMain/kotlin/androidx/compose/ui/focus/FocusOwnerImpl.kt:269]: Does it work with our root focus? Do we have some test for it?

File: libraryversions.toml [hunk 3]
```diff
@@ -20,7 +20,7 @@ CAMERA_TESTING = "1.0.0-alpha01"
 CARDVIEW = "1.1.0-alpha01"
 CAR_APP = "1.7.0-alpha01"
 COLLECTION = "1.4.0"
-COMPOSE = "1.7.0-beta05"
+COMPOSE = "1.7.0-beta06"
 COMPOSE_COMPILER = "1.5.4"
```
Comment (MatkovIvan) [inline libraryversions.toml:24]: I see some changes from the compiler were included. Is this version in sync?

File: compose/ui/ui-graphics/src/androidMain/kotlin/androidx/compose/ui/graphics/layer/AndroidGraphicsLayer.android.kt (hunk 1 above)
Comment (igordmn) [reply compose/ui/ui-graphics/src/androidMain/kotlin/androidx/compose/ui/graphics/layer/AndroidGraphicsLayer.android.kt:765]: Not sure, created a task https://youtrack.jetbrains.com/issue/CMP-5892/Apply-changes-from-1.7.0-beta06-from-AndroidGraphicsLayer.android.kt

File: compose/ui/ui/src/commonMain/kotlin/androidx/compose/ui/focus/FocusOwnerImpl.kt (hunk 2 above)
Comment (igordmn) [reply compose/ui/ui/src/commonMain/kotlin/androidx/compose/ui/focus/FocusOwnerImpl.kt:269]: It should be covered by [ComposeFocusTest](https://teamcity.jetbrains.com/buildConfiguration/JetBrainsPublicProjects_Compose_DesktopTests/4735781?buildTab=tests&name=ComposeFocusTest)

File: libraryversions.toml (hunk 3 above)
Comment (igordmn) [reply libraryversions.toml:24]: We don't use `compiler`, but we usually include it in Compose updates.

But it isn't aligned with Compose, so better to start excluding it.

P.S. we probably need scripts for copy-paste folders as well, to pick the right folders 🫠.

**Review by MatkovIvan — APPROVED** [review]

**Pull request merged**